src/utilities/emailSender.js

```diff
@@ -86,17 +88,23 @@ const closure = () => {
     replyTo = null,
     acknowledgingReceipt = null,
   ) {
-    if (process.env.sendEmail) {
-      queue.push({
-        recipient,
-        subject,
-        message,
-        cc,
-        bcc,
-        replyTo,
-        acknowledgingReceipt,
-      });
-    }
+    return new Promise((resolve, reject) => {
+      if (process.env.sendEmail) {
+        queue.push({
+          recipient,
+          subject,
+          message,
+          cc,
+          bcc,
+          replyTo,
+          acknowledgingReceipt,
+          resolve,
+          reject,
+        });
+      } else {
+        resolve('Email sending is disabled');
```

Comment:
We probably better to just disable the whole queue system instead of resolving each promise in the message queue. IMO, this change noticing email sending is disabled is probably irrelevant to the current scope through.

Reply:
it was necessary to get responses in emailController.js